### PR TITLE
[1LP][RFR]Service Catalogs widgetastic

### PR DESCRIPTION
**Pull request by sshveta**

### CODE REVIEW

**Comment by sshveta** [comment]
Not tested on 5.6 and upstream due to internal error

File: cfme/fixtures/service_fixtures.py [hunk 1]
```diff
@@ -23,7 +23,7 @@ def dialog():
                      tab_label="tab_" + fauxfactory.gen_alphanumeric(), tab_desc="my tab desc",
                      box_label="box_" + fauxfactory.gen_alphanumeric(), box_desc="my box desc")
     service_dialog.create(element_data)
-    return dialog
+    yield service_dialog
```
Comment (izapolsk) [inline cfme/fixtures/service_fixtures.py]: OPTIONAL: it makes sense to add teardown/destructor part if you decided to use yield_fixture

File: cfme/services/catalogs/service_catalogs.py [hunk 2]
```diff
@@ -1,114 +1,128 @@
...
 
     def step(self):
         try:
-            sel.click(list_tbl.find_row_by_cell_on_all_pages({'Name': self.obj.service_name}))
+            self.prerequisite_view.service_catalogs.tree.click_path("All Services",
+                self.obj.catalog.name, self.obj.name)
         except:
             raise NoSuchElementException()
 
-    def resetter(self):
```
Comment (izapolsk) [inline cfme/services/catalogs/service_catalogs.py]: OPTIONAL: are you sure you don't need resetter any longer ?

File: cfme/fixtures/service_fixtures.py (hunk 1 above)
Comment (sshveta) [inline cfme/fixtures/service_fixtures.py]: This needs some testing as deleting dialog depends on other child items. Will include my next PR .

File: cfme/services/catalogs/service_catalogs.py (hunk 2 above)
Comment (sshveta) [inline cfme/services/catalogs/service_catalogs.py]: No , Don't need it anymore.

**Pull request merged**